submariner/centralized-control-plane.md

```diff
@@ -0,0 +1,386 @@
+# Centralized Service Control Plane
+
+## Summary
+
+Current Submariner service management follows the model and guidelines
+ defined in [MCS](https://github.com/kubernetes-sigs/mcs-api). Specifically the
+ current design has these attributes:
+
+- Exported services are automatically reflected into all clusters in the ClusterSet;
+- Service are exported and imported using the same namespace and name;
+- Service exports are managed from workload clusters; and
+- Imported services, having the same namespaced name, are combined on import into one service.
+
+The above works well in an environment where clusters are used by a single
+ administrative domain and services are commonly shared across all clusters.
+ For example, when a company runs clusters as the runtime infrastructure and developers can
+ deploy to any cluster for availability, redundancy, or geographic proximity. We would like
+ to extend this model to support additional use cases. Extensions proposed are enhancement
+ to the base MCS objects and concepts and allow their use in use cases that aren't well
+ supported by the current MCS design. In order to minimize the changes to the base MCS
+ specification, we also outline how a management layer can use the modified definitions
+ to support more use cases as a layer atop MCS.
+
+### Use Cases
+
+We would like to support Multi-Cluster Services in the following additional use cases (see
+also Context section below.):
+
+- **Separate administrative domains**: a deployment model where clusters are owned and operated
+  by different teams, each having independent control and configuration.
+- **Global load balancing**: allow a global load balancer to use services in different clusters.
+- **Non flat networks**: exported services are accessible via an ingress point, such as Ingress,
+  NodePort, LoadBalancer, etc., and not directly using IP addresses of backing Pods.
+- **Centralized management plane**: service sharing is managed from a central location, in a
+  controlled manner (i.e., only identified services in a catalog may be shared).
```

Comment:
These are all outside the scope of Submariner itself, each point for it's own reasons.

Since the section details "We would like to support Multi-Cluster Services in the following additional use cases", this implies that the desired capabilities should be part of Submariner and in it's scope.

Hence it seems to me this proposal as a whole should live outside of submariner, and specific enhancements which fall in Submariner's scope.